.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
+
+test_patterns = ["tests/**"]
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
I would want to do some more looking into this tool before brining it into this repository. I'll check it out when I have a bit more time to actually dig into it. One thing I would want to make sure it that it could be run locally with `tox` without any additional setup, so that we keep the uniformed CI interface.

Reply:
Hi @vimalloc 

Thanks for the review! You can check out the dashboard link that I shared on the PR above. As per my understanding, Tox is a different tool used to run tests. DeepSource is a code quality analysis tool, which is run separately. A CLI is under the progress which would be released in a few months. However DeepSource is a service whose checks will run in parallel to CI. It won't be the part of CI.